shipyard_blueprints/dbt/shipyard_dbt/cli/trigger_and_download.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import sys
+import time
+
+import shipyard_bp_utils as shipyard
+from shipyard_bp_utils.artifacts import Artifact
+from shipyard_templates import ShipyardLogger, Etl, ExitCodeException
+
+from shipyard_dbt import DbtClient
+
+logger = ShipyardLogger.get_logger()
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
```

Comment:
was there no `--wait-for-completion` arg for this before?

Reply:
wait for completion was always set to TRUE in this blueprint. Since it is being moved to its own file wait for completion is not needed.